### ISSUE TITLE
Are changes permanent ?

### ISSUE THREAD

**Issue by postadelmaga** [tiziw/iuvolt#5]
It is not clear to me if the changes to the cpu register are permanent or a reboot is enough to reset the default values.

I think this can be a valuable information to all newbie like me.

**Comment by tiziw** [comment]
Registers are located in internal CPU memory, dedicated for storing values that require direct access by the CPU. The fact that it's memory means it's not persistent after reboot. In this case, the MSR registers will reset on reboot/shutdown (and hibernate) and on sleep. This was briefly mentioned in the linux-intel-undervolt github readme which was linked to in the credits section.

If you set the values too low it could crash your system, however after a reboot the values will reset and it'll work just fine. Make sure you test a certain voltage is safe before you set it on config and enable the boot service.

tl;dr no, but if you enable the systemd service it'll be set automatically after login.
